docs/tests/getting-started/github-actions.yml

```diff
@@ -23,9 +23,20 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v2
 
+      - name: Project init
+        uses: dagger/dagger-for-github@v2
+        # See all options at https://github.com/dagger/dagger-for-github
+        with:
+          args: project init
+
+      - name: Project update
+        uses: dagger/dagger-for-github@v2
+        # See all options at https://github.com/dagger/dagger-for-github
+        with:
+          args: project update
```

Comment:
This should not run in the CI. You are effectively updating all dagger deps and don't know whether they will work or not. This is similar to running `go dep update` in the CI an YOLO-ing. You wouldn't do that with any dependencies, because then there will come a day when things will just break due to dependencies upgrading in an unpredictable way.

Reply:
If you pin your versions, `project update` shouldn't actually update your packages, just install them. I should take another look at how we're doing this. I usually work with two files: 1) for version constraints used to easily upgrade; 2) a lock file with exactly pinned dependencies, to their digest if possible.

Reply:
Great point!

Do you know how we are doing this? 🙂